src/pages/public_registration/PublicRegistrationContributors.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import { Button, IconButton, Link, Typography } from '@material-ui/core';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { useTranslation } from 'react-i18next';
+import AffiliationHierarchy from '../../components/institution/AffiliationHierarchy';
+import OrcidLogo from '../../resources/images/orcid_logo.svg';
+import { Contributor, ContributorRole } from '../../types/contributor.types';
+import { getDistinctContributorUnits } from '../../utils/institutions-helpers';
+import { BookType } from '../../types/publicationFieldNames';
+
+const StyledContributorsGrid = styled.div`
+  display: grid;
+  align-items: start;
+  grid-template-columns: 1fr auto;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    grid-template-columns: 1fr;
+  }
+`;
```

Comment:
TIL: https://css-tricks.com/things-ive-learned-css-grid-layout/#you-cant-have-tetris-shaped-grid-areas

Er nok derfor det fungerte med `"contributors button" "contributors ."` og ikke `"contributors button" "contributors contributors"`

Reply:
mm, hvis noe virker for godt til å være sant, osv